index.js

```diff
@@ -11,6 +11,11 @@ module.exports = {
     "no-eval": 2,
     "no-extend-native": 2,
     "no-extra-bind": 2,
-    "no-floating-decimal": 2
+    "no-floating-decimal": 2,
+
+    // Comma
+    "comma-dangle": ["error", "always-multiline", "only-multiline"],
```

Comment:
How does this config work? Do you want `"always-multiline"` or `"only-multiline"`?

Reply:
I'll check if I didn't make any error. But the idea is that the comma would be mandatory on multi-line and forbidden for one-liners.

Reply:
I probably first got there via only-multiline.

With the always-multiline we have "[...] and disallows [...]". That should indeed be enough.